.github/dependabot.yml

```diff
@@ -0,0 +1,34 @@
+version: 2
+updates:
+  - package-ecosystem: 'npm'
+    directory: '/'
```

Comment:
I don't think this is going to work as-is; most of our repos can use this sort of config because the generally use yarn workspaces, but we intentionally set up the examples repo to *not* use workspaces so the examples will operate more independently.

Workspaces used to be a requirement to make a reasonable dependabot config, but as of quite recently they support [a multi-directory specification](https://github.blog/changelog/2024-06-25-simplified-dependabot-yml-configuration-with-multi-directory-key-directories-and-wildcard-glob-support/) we should try here.